src/lib/style-extends.html

```diff
@@ -25,7 +25,6 @@
       var rules = styleUtil.rulesForStyle(style);
       var self = this;
       styleUtil.forEachStyleRule(rules, function(rule) {
-        var map = self._mapRule(rule);

```

Comment:
This changes seems to have caused a failure on Travis https://travis-ci.org/Polymer/polymer/jobs/88971521#L2290. This line should probably be `self._mapRule(rule)` (note the removed assignment to `map`).

Make sure you run the test suite before submitting a Pull-request. #2658 has a pending explanation on how you can do this.